pages/docs/user-docs/docs-create-an-image.md

```diff
@@ -22,105 +23,136 @@ Binding image to loop
 Creating file system within image
 Image is done: container.img
 ```
+How big is it?
 
-Let's import an operating system into it.
-
-```bash
-singularity import container.img docker://ubuntu:latest
+```
+$ du -sh container.img 
+29M     container.img
 ```
 
-We can now use the command `ls` to look at the files and permissions of the container:
+We can now use `ls` to list details about the image.
 
-```bash
+```
 $ ls -l container.img 
--rwxr-xr-x 1 vanessa vanessa 805306400 Apr  6 19:24 container.img
+-rwxr-xr-x 1 user group 805306400 Apr 15 11:11 container.img
 ```
 
-How big is it?
+Note the permissions of the image make it executable. Singularity images [can be executed directly](/docs-run).
+
+### Creating an image of a different size
+
+You can change the maximum size of an image you create using the `--size` option. Note that `--size` is not a global option.  It is an option to the `create` sub-command and must therefore follow it:
 
-```bash
-$ du -sh container.img 
-172M     container.img
 ```
+$ singularity create --size 2048 container2.img
+Initializing Singularity image subsystem
+Opening image file: container2.img
+Creating 2048MiB image
+Binding image to loop
+Creating file system within image
+Image is done: container2.img
 
-Here we created a new container image called `container.img` in the current directory.
+$ ls -lh container*.img 
+-rwxr-xr-x 1 user group 2.1G Apr 15 11:34 container2.img
+-rwxr-xr-x 1 user group 769M Apr 15 11:11 container.img
+```
 
-Also, notice the permissions of the container image as it is executable. This is important in that Singularity images can be executed directly (as long as Singularity is installed on the host system).
+### Overwriting an image with a new one
 
-You can increase or change the default image size using the --size option to create (option ordering is very important with Singularity and it must follow the 'create' sub-command).
+If you have already created an image and wish to overwrite it, you can do so with the `--force` option.  This option must also follow the `create` sub-command.
+
+```
+$ singularity create --size 512 --force container2.img
+Initializing Singularity image subsystem
+Opening image file: container2.img
+Creating 512MiB image
+Binding image to loop
+Creating file system within image
+Image is done: container2.img
 
+$ ls -lh container*.img 
+-rwxr-xr-x 1 user group 513M Apr 15 11:39 container2.img
+-rwxr-xr-x 1 user group 769M Apr 15 11:11 container.img
+```
 
-## Mounting an image
-Once the image has been created, you can mount it with the following command:
+{% include asciicast.html source='docs-create-create.js' title='How to create images' author='davidgodlove@gmail.com'%}
 
-```bash
-$singularity mount container.img /mnt
+## Increasing the size of an existing image
+You can increase the size of an image after it has been instantiated by using the 'expand' Singularity sub-command as follows:
```

Comment:
I think expand is in the single quote instead of the weird `expand` character, it doesn't render in grey like it should:

![image](https://cloud.githubusercontent.com/assets/814322/25068122/b14c2370-2227-11e7-87c5-8a168cd27058.png)